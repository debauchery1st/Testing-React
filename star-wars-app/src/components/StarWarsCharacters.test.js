import React from "react";
import { render } from "@testing-library/react";
import StarWarsCharacters from "./StarWarsCharacters";
import renderer from "react-test-renderer";

test("renders", () => {
  const tst = renderer.create(<StarWarsCharacters />);
  let el = tst.toJSON();
  expect(el).toMatchSnapshot();
});
