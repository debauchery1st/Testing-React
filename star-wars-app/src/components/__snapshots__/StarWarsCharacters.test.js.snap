// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    className="buttons"
  >
    <button
      disabled={true}
      onClick={[Function]}
    >
      Previous
    </button>
    <button
      disabled={true}
      onClick={[Function]}
    >
      Next
    </button>
  </div>
</div>
`;
